src/plugins/intel_gpu/src/kernel_selector/cl_kernels/rms_gpu_ref.cl

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/batch_headers/fetch_data.cl"
+
+KERNEL(rms_gpu_ref)(
+    OPTIONAL_SHAPE_INFO_ARG
+    const __global INPUT0_TYPE* input,
+    const __global INPUT1_TYPE* gamma,
+    __global OUTPUT_TYPE* output)
+{
+    const uint b = get_global_id(0);
+    const uint f = get_global_id(1);
+
+    ACCUMULATOR_TYPE rms = ACCUMULATOR_VAL_ZERO;
+    for (uint z = 0; z < INPUT0_SIZE_Z; z++)
+    {
+        for (uint y = 0; y < INPUT0_SIZE_Y; y++)
+        {
+            for (uint x = 0; x < INPUT0_SIZE_X; x++)
+            {
+#if INPUT0_DIMS == 4
+                const uint input_idx = INPUT0_GET_INDEX(b, f, y, x);
+#elif INPUT0_DIMS == 5
+                const uint input_idx = INPUT0_GET_INDEX(b, f, z, y, x);
```

Comment:
I think it can be replaced with `get_input_index` call from `fetch_utils.cl`

Reply:
Updated to use `get_input|output_index` inline function